src/cfnlint/decode/cfn_json.py

```diff
@@ -265,10 +260,11 @@ def CfnJSONObject(s_and_end, strict, scan_once, object_hook, object_pairs_hook,
             raise JSONDecodeError('Duplicate found {}'.format(err), s, begin, key)
         except NullError as err:
             raise JSONDecodeError('Null Error {}'.format(err), s, begin, key)
-        return result, end
+
```

Comment:
You've removed the return from within this object_pairs_hook block -- is that intended? I noticed the return remains in an early block around line 192.

Reply:
it was kind of intentional but not done well.  I moved all the conversion to the dict_node class to the object_pairs_hook and then added this back.  It should be now that everything is getting converted to our class of dict_node and returned using that return.